src/msbuild/NuGet.config

```diff
@@ -15,13 +15,13 @@
     <add key="dotnet9" value="https://dnceng.pkgs.visualstudio.com/public/_packaging/dotnet9/nuget/v3/index.json" />
     <add key="dotnet9-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet9-transport/nuget/v3/index.json" />
     <add key="vs-impl" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl/nuget/v3/index.json" />
-    <!-- We don't want other than VS OpenTelemetry libraries from vs-impl -->
-    <packageSourceMapping>
-      <packageSource key="vs-impl">
-        <package pattern="Microsoft.VisualStudio.OpenTelemetry*" />
-      </packageSource>
-    </packageSourceMapping>
   </packageSources>
+  <!-- We don't want other than VS OpenTelemetry libraries from vs-impl -->
+  <packageSourceMapping>
```

Comment:
Isn't this going to fail the build because there are no wild cards for the other packages?

Reply:
Good point - I will add `*` mappings for all other sources.

Reply:
Fixed with https://github.com/dotnet/dotnet/pull/550/commits/d55a2e0d17c2b56243c60f2a23f389e4ecdb51fe